proto/info/patch_envelope.proto

```diff
@@ -55,28 +45,15 @@ message ZInfoPatchEnvelope {
     repeated string errors = 6;
 }
 
-message ZInfoOpaqueAppInstanceStatus {
-    string appId = 1;
-    string appName = 2;
-    EVE_OPAQUE_STATUS_ACTION opaqueStatusAction = 3;
-    bytes opaqueStatus = 4;
-}
-
 // Provide information on PatchEnvelope usage per app
 message ZInfoPatchEnvelopeUsage {
-    // count the number of times app instance called patch APIs
-    uint64 patchApiCallCount = 1;
-    // information about patchEnvelope referred to this app
-    ZInfoPatchEnvelopeApp patchStatus = 2;
-}
-
-// Statistics on usage of specific patchEnvelope for specific app
-message ZInfoPatchEnvelopeApp {
     // UUID of PatchEnvelope
     string uuid = 1;
     // Version of PatchEnvelope
     string version = 2;
-    // Number of times this PatchEnvelope
-    // was sucessfully downloaded
-    uint64 timesDownloaded = 3;
+    // count the number of times app instance called patch APIs
+    uint64 patchApiCallCount = 3;
+    // count the number of times app instance actually downloaded
+    // whole patch envelope or part of it
+    uint64 timesDownload = 4;
```

Comment:
Apart from breaking changes, yetus suggests to change this to snake case. I didn't know that we are using snake case in proto messages

Reply:
Yes, we are using snake case in new protobuf (unless it is new values in an existing enum when we follow the existing convention for that enum).

The protoc produces the same output i.e., lowerCamelCase even as you change the proto file to use lower_snake_case.